modules/core/src/main/scala/lucuma/odb/api/model/time.scala

```diff
@@ -9,37 +9,36 @@ import eu.timepit.refined.api.Validate.Plain
 import eu.timepit.refined.boolean.{And, Not}
 import eu.timepit.refined.numeric.{Greater, GreaterEqual, Interval, Less, NonNegative}
 import io.circe.Decoder
+import lucuma.core.syntax.time._
 import shapeless.Nat._0
 
-import java.time.{LocalDate, Month}
-import java.util.concurrent.TimeUnit
+import java.time.{Duration, LocalDate, Month}
 
-import scala.concurrent.duration._
 import scala.util.Try
 
 object time {
 
-  implicit val nonNegativeFiniteDurationValidate: Plain[FiniteDuration, GreaterEqual[_0]] =
+  implicit val nonNegativeDurationValidate: Plain[Duration, GreaterEqual[_0]] =
     Validate.fromPredicate(
-      (d: FiniteDuration) => d.length >= 0L,
-      (d: FiniteDuration) => s"$d is not negative",
+      (d: Duration) => d.toNanos >= 0L,
+      (d: Duration) => s"$d is not negative",
       Not(Less(shapeless.nat._0))
     )
 
-  type NonNegativeFiniteDuration = FiniteDuration Refined NonNegative
+  type NonNegativeDuration = Duration Refined NonNegative
```

Comment:
We discussed unifying the two instances that I mistakenly created.  I can do it in another PR.

Reply:
Actually, this is the unified instance. 😄 What remains in `gen/package.scala` is syntax specifically for the `AcqExposureTime` and other 2 types in that file. But, now they make use of the refined type in this file instead of redefining it.